src/main/java/org/webshar/hrms/model/db/OrganizationLeave.java

```diff
@@ -0,0 +1,74 @@
+package org.webshar.hrms.model.db;
+
+import javax.persistence.Entity;
+import java.io.Serializable;
+import java.util.Date;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+@Entity
+@Table(name = "organization_leave", uniqueConstraints = {@UniqueConstraint(columnNames = {"id"})})
+public class OrganizationLeave implements Serializable
+{
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "id", nullable = false, unique = true, length = 11)
+  @Getter
+  @Setter
+  private Long id;
+
+  @Column(name = "leave_type_id", nullable = false, unique = true, length = 11)
+  @Getter
+  @Setter
+  private Long leaveTypeId;
+
+  @Column(name = "org_id", nullable = false, unique = true, length = 11)
+  @Getter
+  @Setter
+  private Long organizationId;
```

Comment:
instead of maintaining ids, use actual objects and give @OneToOne mapping.
e.g. 
```
class OriganizationLeave {
@OneToOne LeaveType leaveType,
@OneToOne Organization organization
} 
```
Internally it will create only id mapping at table level.
Take a similar approach for rest and upcoming model designs as well.